common/src/msgs/data_requests/types.rs

```diff
@@ -244,6 +244,8 @@ pub struct DrConfig {
     /// This is the delay before the backup executors are allowed to start
     /// executing the data request.
     pub backup_delay_in_blocks:   NonZero<u64>,
+    /// The maximum size of all the reveals in a data request.
```

Comment:
Let's add the unit in the name or at least in the comment right?
Looking below, it seems to be bytes.
```suggestion
    /// The maximum size in bytes of all the reveals in a data request.
```